docs/content/docs/data-structures/overview.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Overview of Distributed Data Structures
+menuPosition: 1
+---
+
+The Fluid Framework provides developers with distributed data structures (DDSes) that automatically ensure that each
+connected client has access to the same state. The APIs provided by DDSes are designed to be familiar to programmers
```

Comment:
> each connected client has access to the same state
**eventually**, right? Otherwise it reads as if every operation is blocking.

Reply:
Good point